docs/videos/healthcare-and-life-sciences/README.md

```diff
@@ -127,15 +125,41 @@ storage intact and b) you can build software before you deploy your cluster.
 
 1. Deploy the `setup` group
 
-   The next `ghpc` prompt will again ask you to display, apply, stop, or
-   continue without applying the `setup` group. Select apply.
+   The next `ghpc` prompt will ask you to **display**, **apply**, **stop**, or
+   **continue** without applying the `setup` group. Select 'apply'.
 
    This group will create a network and file systems to be used by the cluster.
 
    > [!NOTE]
    > At this point do not proceed with the ghpc prompt for the `cluster` group.
    > Continue with the steps below before proceding.
 
+   This step will create a storage bucket for depositing software. The bucket
+   will have the prefix `hcls-user-provided-software` followed by a the project
+   id (e.g. `hcls-01`) and a random suffix, for example
+   `hcls-user-provided-software-hcls-01-34c8749a`.
+
+   Here are three ways to locate the bucket name:
+
+   1. The output of ghpc
```

Comment:
Remove option 1 & 2. These are kind of flaky ways of finding the name. If we actually want to provide a definitive way for the user to look up the name, add it to the outputs of the module that creates the bucket. This will cause the name to be printed on apply. Then the instructions can reference that line.